hooks/tk-multi-breakdown2/basic/scene_operation.py

```diff
@@ -121,16 +121,34 @@ def update(self, item):
         self.logger.info("Updating %s (%s)" % (node_name, node_type))
         sg_data = item["sg_data"]
         if not sg_data or not sg_data.get("path", {}).get("local_path", None):
+            self.logger.warning("Invalid SG data %s, not updating" % sg_data)
             return False
-        path = sg_data["path"]["local_path"]
-        node_id = item["extra_data"]["node_id"]
-        engine = self.parent.engine
-        current_project = engine.current_project
-        # For now just update the path in place.
-        clip = current_project.get_clip_by_id(node_id)
-        if not clip:
-            self.logger.warning("Unable to retrieve a clip with id %s" % node_id)
+        try:
+            path = sg_data["path"]["local_path"]
+            node_id = item["extra_data"]["node_id"]
+            engine = self.parent.engine
+            current_project = engine.current_project
+            import_in_bin_template = self.parent.get_template("import_in_bin_template")
+            if import_in_bin_template:
+                import_bin_path = import_in_bin_template.apply_fields(sg_data)
```

Comment:
any worries about windows support?
it seems here that this could cause problems since `ensure_bins_for_path` expects `/` as a separator?

Reply:
it's a "path" in the app... Not in the filesystem...